### ISSUE TITLE
Removing property definitions from main specification text?

### ISSUE THREAD

**Issue by rartino** [Materials-Consortia/OPTIMADE#505]
Now that #445 is merged, should we remove the property definitions from `optimade.rst`? It does not seem ideal that we have two set of "competing" definitions that could diverge going forward. Technically it is of course possible to insert a link in that part of the document to what we consider the single-point-of-truth of property definitions. However, that cannot be `https://schemas.optimade.org/`, since we need to be able to edit and reivew edits of the property definitions while we work.

In discussion with @ml-evs, one idea is that we actually check into the repo a generated markdown document for the "OPTIMADE Standard" from the property definition framework that as a single document contains all the property definitions in the same style as before.

**Comment by merkys** [comment]
I agree with the removal of actual property definitions. AFAIK, they do not contain anything which is not in #445, and there should be a single source of truth.

> In discussion with @ml-evs, one idea is that we actually check into the repo a generated markdown document for the "OPTIMADE Standard" from the property definition framework that as a single document contains all the property definitions in the same style as before.

Sounds great - a human-readable description generated from the property definition framework makes a lot of sense.

**Comment by merkys** [comment]
This is however blocked by the pending PRs introducing or modifying already existing properties. I would suggest resolving them first, or reworking them into PRs modifying property definitions directly.
